src/eth/eip_1559_transaction.rs

```diff
@@ -61,6 +61,42 @@ impl EthEip1559TxArgs {
 
         Ok(self)
     }
+
+    pub fn rlp_signed_message(&self) -> anyhow::Result<Vec<u8>> {
+        let mut stream = rlp::RlpStream::new_list(12);
```

Comment:
Is there a less scary API that doesn't require changing this number on a new field?

Reply:
Switched to the unbounded API